closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,44 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0, optionsFactoryCalled = 0, callbackInvoked = 0;
+  var wrapperFactory = new goog.net.WrapperXmlHttpFactory(
+      function() {
+        xhrFactoryCalled++;
+        return goog.net.XmlHttp();
+      },
+      function() {
+        optionsFactoryCalled++;
+        return {};
+      });
+
+  sendInstances = goog.testing.net.XhrIo.getSendInstances();
+  var returnedXhr = goog.testing.net.XhrIo.send(
+      'url', undefined, 'GET', '', undefined, 0, false, wrapperFactory);
+  assertEquals('sendInstances_ after send', 1, sendInstances.length);
+  xhr = sendInstances[sendInstances.length - 1];
+  assertTrue('isActive after request', xhr.isActive());
+  assertEquals(returnedXhr, xhr);
+  assertEquals(
+      'readyState after request', goog.net.XmlHttp.ReadyState.LOADING,
+      xhr.getReadyState());
+
+  xhr.simulateResponse(200, '');
+  assertFalse('isActive after response', xhr.isActive());
+  assertEquals(
+      'readyState after response', goog.net.XmlHttp.ReadyState.COMPLETE,
+      xhr.getReadyState());
+
+  xhr.simulateReady();
+  assertEquals('sendInstances_ after READY', 0, sendInstances.length);
+
+  // check if all wrapperFactory is used during send
```

Comment:
[Complete sentences should use appropriate capitalization and punctuation.](https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments)